vcpkg/commands/edit.md

```diff
@@ -0,0 +1,56 @@
+---
+title: vcpkg edit
+description: Command-line reference for the vcpkg edit command. Opens a port's folder in a new code editor window.
+author: vicroms
+ms.author: viromer
+ms.date: 7/24/2023
+ms.prod: vcpkg
+---
+# vcpkg edit
+
+## Synopsis
+
+```Console
+vcpkg edit <ports> [--options]
+```
+
+## Description
+
+Opens a port's corresponding `ports` folder in a new _Visual Studio Code_ window.
```

Comment:
We should probably add description covering the circumstances where this command works and doesn't work. For example, I don't know what this command does in the case of custom git or filesystem registries -- off the top of my head, I would assume it actually does work for filesystem registries but gives bogus results for git registries.